src/types/json.h

```diff
@@ -211,6 +211,17 @@ struct JsonValue {
     return types;
   }
 
+  Status Toggle(std::string_view path) {
+    try {
+      jsoncons::jsonpath::json_replace(value, path, [&](const std::string & /*path*/, jsoncons::json &val) {
+        if (val.is_bool()) val = !val.as_bool();
```

Comment:
> JSON.TOGGLE returns an array of integer replies for each path, the new value (0 if false or 1 if true), or nil for JSON values matching the path that are not Boolean. 

The return value should be an array of integers. That said if there are two paths matched and one of them is NOT a boolean value, then it should return an array with:

1) 0(or 1)
2) nil

Reply:
Thanks for poiniting out. will fix this

Reply:
Thank you!